src/app/campaign/availability/availability.component.spec.ts

```diff
@@ -24,31 +24,31 @@ describe('AvailabilityComponent', () => {
       {
         startDate: new Date('2019-10-01'),
         endDate: new Date('2019-10-05'),
```

Comment:
The component now expects that the availability numbers include the allocated amount. Previously the rollups were just doing weekly and totals and letting the component display availability as allocated + availability